docs/paas-plus/dev/platform-as-a-service-plus.md

```diff
@@ -0,0 +1,161 @@
+**Why?** At the moment, we only support upgrades on repositories using GitHub or GitLab. If your project uses a different git provider, please reach out to your CSM to ensure we can incorporate it in our product roadmap.
+
+**6. Customer should be headless (optional)**
+**Why?** Currently, the Spryker quality gate does not check compliance of FE customizations with Spryker best practices. You can either move to headless or apply frontend upgrades manually.
+
+//needs better formating
+Disclaimer: As we evolve the product, we will be working on extending functionalities and pre-requirements will be adjusted accordindly.
+
+### Assumptions and Exceptions
+We assume customers are already using Spryker PaaS Cloud services. When not applicable, customer needs to migrate to our Cloud service.
+
+
+
+## Step 2: Onboarding to Spryker CI
+Once you finalize step 1, you will be onboarded to Spryker CI and benefit from Spryker quality gate and automated upgrades.
+
+### What do you need to provide?
+In order to get you started, please provide the email of your Spryker Cloud Admin to your CSM. Your Spryker Cloud Admin user will be a Super User, which mean they will have RW rights on Spryker CI (e.g. ability to change repository; ability to invite new users to their workspace).
+Once we receive your user email, we will provision a Spryker CI workspace for you. You will then receive the invitation to your Spryker CI workspace via email.
+
+<img width="831" alt="image2" src="https://user-images.githubusercontent.com/83701393/176354827-ff1d3378-9a3b-4e15-a51c-c7bc6b56b5cf.png">
+
+
+## 5 steps to get started with Spryker CI
+
+**Step 1. Go to your $(workspacename) workspace**
+You receive an email with a one-time invitation token, but your invitation is always accessible by signing in on Buddy.works. Create your account & accept the invitation to your workspace.
+<img width="496" alt="image3" src="https://user-images.githubusercontent.com/83701393/176354843-956e46fe-4eb7-407b-b33a-dc6aa11eb38b.png">
+
+
+**Step 2. View projects inside your workspace**
+Your Spryker CI workspace contains the Spryker Upgrader Service project, as well as a project for each of your environments (e.g. development, production, etc).
+<img width="424" alt="Image4" src="https://user-images.githubusercontent.com/83701393/176354875-7333d656-f659-4aae-b650-803d7309b5c6.png">
+
+
+**Step 3. Invite new users***
+*Feature only available for Super Users.
+
+Add new users to your project and assign user permissions. Two roles are available: Super Users (RW) and Regular Users (RO & ability to run pipelines).
+
+
+**Step 4. Connect your repository***
+*Feature only available for Super Users.
+
+Connect your repository to a project, before running the pipelines. Open the“Code” section, click on the kebab menu and select "Switch repository or Git provider”. 
+
+![gif1](https://user-images.githubusercontent.com/83701393/176354903-a9d0a669-af3f-449d-bde7-cfc696e96569.gif)
```

Comment:
![gif1](https://user-images.githubusercontent.com/83701393/176374248-6939da08-157c-431c-8101-69251734307b.gif)

Reply:
Do we have a higher resolution gif?

Reply:
hmmm.... I recorded with quicktime.. I can try again, but not sure it will be much better..